Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -980,6 +1035,8 @@ void AODProducerWorkflowDPL::init(InitContext& ic)
     mTrackCovDiag = 0xFFFFFFFF;
     mTrackCovOffDiag = 0xFFFFFFFF;
     mTrackSignal = 0xFFFFFFFF;
+    mTrackTime = 0xFFFFFFFF;
+    mTrackTimeError = 0xFFFFFFFF;
```

Comment:
Why do we have different defaults here than above? (`-999`)

Reply:
sorry, did not understand the question: the lines you refer to are compression masks (when no compression is requested, what they have to do with -999?)

Reply:
Sorry, forget about it.